src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -490,7 +494,17 @@ static ConfigDef newConfigDef() {
             ERROR_GROUP,
             2,
             ConfigDef.Width.NONE,
-            ERRORS_DEAD_LETTER_QUEUE_TOPIC_NAME_DISPLAY);
+            ERRORS_DEAD_LETTER_QUEUE_TOPIC_NAME_DISPLAY)
+        .define(
+            SNOWFLAKE_CLEANER_MAX_RETRIES_CONFIG,
+            Type.LONG,
```

Comment:
nit: int should be enough?

Reply:
true, no strong preference. can change it to int